driver/ppm_fillers.c

```diff
@@ -1390,6 +1391,17 @@ int f_proc_startupdate(struct event_filler_arguments *args)
 		/* Parameter 26: exe_file mtime (last modification time, epoch value in nanoseconds) (type: PT_ABSTIME) */
 		res = val_to_ring(args, mtime, 0, false, 0);
 		CHECK_RES(res);
+
+		/* Parameter 27: uid */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 5, 0)
+		euid = from_kuid_munged(current_user_ns(), current_euid());
+#elif LINUX_VERSION_CODE > KERNEL_VERSION(2, 6, 20)
+		euid = current_euid();
+#else
+		euid = current->euid;
+#endif
```

Comment:
do we support kernel versions lower than `2.6.20` :( ?

Reply:
Apparently not 😅 I probably copy-pasted it from somewhere

Reply:
While I'm personally happy to drop 2.6.20 support, let's do it across the whole code, in a separate PR. I do hope we don't have to support < 2.6.20, since even RHEL6 is on 2.6.32

Reply:
@gnosek could we double-check this and cleanup if we don't support <2.6.20?